packages/suite/src/types/wallet/formDraft.ts

```diff
@@ -0,0 +1,4 @@
+import { FormDraftPrefixKeyValues } from '@wallet-constants/formDraft';
```

Comment:
we already have ./types/wallet/form.ts` declarations shared between sendform and invity code, can we put it there?

Reply:
Fixed.